R/method-differentialAbundance.R

```diff
@@ -230,6 +235,7 @@ setMethod("maaslin", signature("AbundanceData", "Comparator"), function(data, co
 #' @param data AbsoluteAbundanceData object
 #' @param comparator Comparator object specifying the variable and values or bins to be used in dividing samples into groups.
 #' @param method string defining the the differential abundance method. Accepted values are 'DESeq' and 'ANCOMBC'.
+#' @param pValueFloor numeric value that indicates the smallest p value that should be returned. The corresponding adjusted p value floor will also be updated based on this value. The default value uses the P_VALUE_FLOOR=1e-200 constant defined in this package.
```

Comment:
This could better describe the relationship between the pvalue floor, adjusted pvalue floor.